src/main/java/pl/agawesolowska/ticketbookingapp/model/dto/BookingRequestDTO.java

```diff
@@ -0,0 +1,29 @@
+package pl.agawesolowska.ticketbookingapp.model.dto;
+
+import java.util.Set;
+
+import javax.validation.constraints.NotNull;
+
+import lombok.Getter;
+import lombok.Setter;
+import pl.agawesolowska.ticketbookingapp.model.entity.Customer;
+import pl.agawesolowska.ticketbookingapp.model.entity.Screening;
+import pl.agawesolowska.ticketbookingapp.model.entity.Seat;
+
+/**
+ * @author Aga Wesołowska
+ *
+ */
+@Getter @Setter
```

Comment:
dlaczego w jednej linii obie adnotacje? to zaciemnia

Reply:
Jeśli sama pracuję nad projektem i mogę wybrać sposób formatowania, to wolę umieścić obie te adnotacje w jednej linii, wtedy od razu widzę czy są tylko gettery czy dodałam również settery. Gettery i settery często dodaje się z automatu, ale ja nad koniecznością setterów zawsze się zastanawiam.